### PR TITLE
Changes from the last course run at Zendesk

### PR DESCRIPTION
**Pull request by lukestephenson-zendesk**
- Dependency bumps (of which the biggest impact is cats-effect upgrade)
- recommending java 11 over java 8 (next PR will recommend java 17)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,17 +6,17 @@
...
-brew cask install homebrew/cask-versions/adoptopenjdk8
+brew tap AdoptOpenJDK/openjdk
+brew cask install adoptopenjdk11
 ```
 
 2. Before the course, please run the following:
 
 ```
-git clone git@github.com:realestate-com-au/applied-scala.git
+git clone git@github.com:zendesk/applied-scala.git
```
Comment (george-wilson-rea) [inline README.md:19]: Please revert this line.

```suggestion
git clone git@github.com:realestate-com-au/applied-scala.git
```

File: build.sbt [hunk 2]
```diff
@@ -2,16 +2,16 @@ name := "applied-scala"
...
-val circeVersion = "0.13.0"
-val http4sVersion = "0.21.6"
-val logbackVersion = "1.2.3"
+val catsVersion = "2.7.0"
+val circeVersion = "0.14.1"
+val http4sVersion = "0.23.11"
+val logbackVersion = "1.2.11"
 val postgresqlVersion = "42.2.4"
-val doobieVersion = "0.8.8"
+val doobieVersion = "1.0.0-RC1"
```
Comment (george-wilson-rea) [inline build.sbt:14]: Could this be a later version like `1.0.0-RC7`? or is that part of a later PR?

**Comment by george-wilson-rea** [comment]
Thank you all so much for contributing these changes!

File: build.sbt (hunk 2 above)
Comment (lukestephenson-zendesk) [reply build.sbt:14]: I've got a later PR to bump a bunch of additional dependencies.

**Review by pat-stockwell — APPROVED** [review]
Love it! Thanks Luke ❤️

**Pull request merged**